erdos/op.py

```diff
@@ -86,14 +92,22 @@ def spin(self):
     def log_event(self, processing_time, timestamp, log_message=None):
         pass
 
+    def log_streams(self, stream_uid, msg):
+        self.loggers[stream_uid].info(msg)
```

Comment:
I think we want to `pickle.dump()` here instead of `.info()` so that we can replay the stream we log.